src/lib.rs

```diff
@@ -1,609 +1,14 @@
-use std::collections::VecDeque;
```

Comment:
do we need this file anymore? seems mostly empty now

Reply:
Ya, we do. It serves as the entrypoint into the lib, kinda like a `main.go` does for a go binary.